CMakeLists.txt

```diff
@@ -160,6 +168,10 @@ if (CUDA_FOUND)
 else()
   set(CUDA_LIB_DEPS "")
   add_definitions(-DCPU_ONLY)
+  # if (USE_TENSORRT)
+  #   message(STATUS "Could not find CUDA, desabling TensorRT backend")
```

Comment:
I believe we may want to generate an error and stop the build if `USE_TENSORRT` is `ON` and CUDA cannot be found ?

Reply:
ok, done